src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractImportCompletionCacheServiceFactory.cs

```diff
@@ -15,10 +15,10 @@ namespace Microsoft.CodeAnalysis.Completion.Providers.ImportCompletion
     internal abstract class AbstractImportCompletionCacheServiceFactory<TProjectCacheEntry, TMetadataCacheEntry> : IWorkspaceServiceFactory
     {
         private readonly ConcurrentDictionary<string, TMetadataCacheEntry> _peItemsCache
-            = new ConcurrentDictionary<string, TMetadataCacheEntry>();
+            = new();
 
         private readonly ConcurrentDictionary<ProjectId, TProjectCacheEntry> _projectItemsCache
-            = new ConcurrentDictionary<ProjectId, TProjectCacheEntry>();
+            = new();
```

Comment:
Consider combining each of these in one line as it's much shorter now.

Reply:
It a bit of a shame we can't do 
```csharp
private readonly new ConcurrentDictionary<ProjectId, TProjectCacheEntry>() _projectItemsCache ;
```
like vb.
```vbnet
private readonly _projectItemsCache new ConcurrentDictionary<ProjectId, TProjectCacheEntry>();
```

Reply:
@AdamSpeight2008 

If C# allowed:

```csharp
private readonly new ConcurrentDictionary<ProjectId, TProjectCacheEntry>() _projectItemsCache ;
```

it will be very confusing. How can you tell if "new" is shadowing a base member or used for instantiation? (for this specific case, it's not shadowing because it's private. But what if it wasn't private)

But I see it's very neat with target-typed new:

```csharp
private readonly ConcurrentDictionary<ProjectId, TProjectCacheEntry> _projectItemsCache = new();
```

Reply:
Using `override` and `virtual` or both `override virtual`
```c#
public overrides virtual readonly new ConcurrentDictionary<ProjectId, TProjectCacheEntry>() _projectItemsCache ;
```